corehq/apps/settings/forms.py

```diff
@@ -142,16 +141,8 @@ def clean_token(self):
 
 class HQTwoFactorMethodForm(MethodForm):
 
-    def __init__(self, *, allow_phone_2fa, **kwargs):
-        super().__init__(**kwargs)
-        if not allow_phone_2fa:
-            # Block people from setting up the phone method as their default
```

Comment:
Was this block moved somewhere else? From my naive perspective it appears to have been removed, which doesn't seem right.

Reply:
See 4ccbb9f9c74ee16ae5179cd22299b53e1dbdcabf